helm/cluster-autoscaler-app/templates/deployment.yaml

```
@@ -106,4 +108,8 @@ spec:
       volumes:
         - name: ssl-certs
           hostPath:
+            {{- if ((.Values.node).caBundlePath) }}
+            path: "{{ .Values.node.caBundlePath }}"
+            {{- else }}
             path: "/etc/ssl/certs/ca-certificates.crt"
+            {{- end }}
```

```suggestion
            path: {{ .Values.node.caBundlePath }}
```

As you set `/etc/ssl/certs/ca-certificates.crt` as the default value in the `values.yaml`, you can also just reference it here directly. No need to hard-code.

---

True, I forgot that!